<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>second</class>
 <widget class="QWidget" name="second">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>414</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>232</green>
        <blue>232</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>232</red>
        <green>232</green>
        <blue>232</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string> Enter A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="aBound"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Enter B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bBound"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>   Enter step</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="step"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>251</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>PID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> Ім'я </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Стан</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Пріоритет</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Час виконання</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="returnButt">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>320</y>
     <width>701</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addTabProc">
      <property name="text">
       <string>Add tab process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addProc">
      <property name="text">
       <string>Add fibonacci process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addProcProt">
      <property name="text">
       <string>Add proteus process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addProcTracert">
      <property name="text">
       <string>Add tracert process</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>561</width>
     <height>53</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="terminateButt">
      <property name="text">
       <string>Terminate process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="suspendButt">
      <property name="text">
       <string>Suspend process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resumeButt">
      <property name="text">
       <string>Resume process</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="priorButt">
      <property name="text">
       <string>Change prior process</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="comboBoxPrior">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>REALTIME_PRIORITY_CLASS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HIGH_PRIORITY_CLASS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ABOVE_NORMAL_PRIORITY_CLASS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NORMAL_PRIORITY_CLASS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BELOW_NORMAL_PRIORITY_CLASS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IDLE_PRIORITY_CLASS</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
